colossalai/kernel/triton/__init__.py

```diff
@@ -9,7 +9,9 @@
 # There may exist import error even if we have triton installed.
 if HAS_TRITON:
     from .context_attn_unpad import context_attention_unpadded
-    from .flash_decoding import flash_decoding_fwd
+    from .flash_decoding import flash_decoding_attention
+    from .flash_decoding_utils import FDIntermTensors
```

Comment:
where is this needed ? I do not see it in kernel

Reply:
It could be optionally used in llama modeling (the way to store intermediate tensors is to be discussed)

Reply:
I don't actually get the usage of this class,  I guess you need to modify modeling?

Reply:
Yeah, this class could be used in engine/kv cache manager/batch state, and finally the tensors will be passed in attention forward in modeling